lib/bunch/macro.ex

```diff
@@ -82,4 +82,17 @@ defmodule Bunch.Macro do
 
     ast
   end
+
+  @doc """
+  Takes a code block, expands macros inside and pretty prints it.
+  """
+  defmacro peek_code(do: block) do
+    Macro.prewalk(block, fn tree -> Macro.expand(tree, __ENV__) end)
```

Comment:
You probably should pass `__CALLER__` instead of `__ENV__`, so that imports/aliases/etc were propagated to the block

Reply:
Yup

Reply:
You can also move this line to a separate function (`expand_deep(ast, env)` ?), it may be useful

Reply:
done